frontend/app/forms/petition-for-dismissal-or-acquittal/petition-for-dismissal-or-acquittal.pdf.component.js

```diff
@@ -1,5 +1,195 @@
+          dataKey="person.homePhone"
+          left="11.65%"
+          top="24.5%"
+        />
+        <PositionedString dataKey="person.email" left="11.65%" top="28.1%" />
+        <PositionedCheckmark
+          dataKey="person.petitionerRepresentative"
+          left={petRepLeft}
+          top="31.6%"
+          shouldShow={data.person.petitionerRepresentative === "petitioner"}
+        />
+        <PositionedCheckmark
+          dataKey="person.petitionerRepresentative"
+          left={petRepLeft}
+          top="33.1%"
+          shouldShow={data.person.petitionerRepresentative === "attorney"}
+        />
+        <PositionedString
+          dataKey="person.petitionerBarNumber"
+          left={barNumberLeft}
+          top="33%"
+        />
+        <PositionedCheckmark
+          dataKey="case.courtType"
+          left="37.5%"
+          top="36.9%"
+          shouldShow={data.case.courtType === "District"}
+        />
+        <PositionedCheckmark
+          dataKey="case.courtType"
+          left="47.9%"
+          top="36.9%"
+          shouldShow={data.case.courtType === "Justice"}
+        />
+        <PositionedString dataKey="case.courtAddress" left="27.8%" top="43%" />
+        <PositionedString left="25.9%" top="39.55%" debugKey="judicialDistrict">
+          {data.case.courtAddress &&
+            getJudicialDistrict(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+        <PositionedString left="50.7%" top="39.55%" debugKey="county">
+          {data.case.courtAddress &&
+            getCounty(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+        <PositionedString debugKey="petitionerName" left="11.9%" top="52.9%">
+          {`${renderData("person.firstName")} ${renderData(
+            "person.middleName"
+          )} ${renderData("person.lastName")}`}
+        </PositionedString>
+        <PositionedString dataKey="case.caseNumber" left="54.1%" top="52.9%" />
+        <PositionedString dataKey="case.judgeName" left="54.1%" top="57.5%" />
+        <PositionedString dataKey="case.caseNumber" left="17.4%" top="65.5%" />
+        <PositionedString
+          dataKey="case.arrestedDate"
+          left="47.4%"
+          top="63.25%"
+        />
+        <PositionedString
+          dataKey="case.lawEnforcementAgencysFileNumber"
+          left="49.75%"
+          top="68.25%"
+        />
+        <PositionedString
+          dataKey="case.resultOfArrest"
+          left="20.55%"
+          top="74%"
+        />
+        <PositionedCheckmark
+          dataKey="case.wasNotConvicted"
+          left="22.5%"
+          top="79.8%"
+        />
+        <PositionedCheckmark
+          dataKey="case.atLeast30DaysSinceArrest"
+          left="22.5%"
+          top="82.4%"
+        />
+        <PositionedCheckmark
+          dataKey="case.noArrestSinceThisArrest"
+          left="22.5%"
+          top="84.86%"
+        />
+        {data.case.dismissalStatus !== "" && (
```

Comment:
I think this could be replaced with just a truthiness check. `data.case.dismissalStatus &&`